site/src/theme/site.css

```diff
@@ -510,4 +516,8 @@ div.code-toolbar>.toolbar button:before {
 
 .nameContainer .anchor:target+h4 {
   background-color: inherit;
-}
\ No newline at end of file
+}
+
+.container {
+  max-width: 1140px;
+}
```

Comment:
This overrides the `max-width` for all of the breakpoints - making it less responsive.  Did you mean something like 
```css
@media (min-width: 1400px) {
  .container {
    max-width: 1140px;
  }
}
```

Reply:
That was my intention, thanks.